src/main/java/teammates/ui/controller/InstructorFeedbackEditPageData.java

```diff
@@ -148,6 +140,28 @@ private void buildExistingQuestionForm(String feedbackSessionName,
         qnForms.add(qnForm);
     }
 
+    private FeedbackQuestionFeedbackPathSettings configureFeedbackPathSettings(
+            FeedbackQuestionAttributes question) {
+        FeedbackQuestionFeedbackPathSettings settings = new FeedbackQuestionFeedbackPathSettings();
+        settings.setSelectedGiver(question.giverType);
+        settings.setSelectedRecipient(question.recipientType);
+
+        boolean isNumberOfEntitiesToGiveFeedbackToChecked =
+                question.numberOfEntitiesToGiveFeedbackTo != Const.MAX_POSSIBLE_RECIPIENTS;
+        settings.setNumberOfEntitiesToGiveFeedbackToChecked(isNumberOfEntitiesToGiveFeedbackToChecked);
+        settings.setNumOfEntitiesToGiveFeedbackToValue(isNumberOfEntitiesToGiveFeedbackToChecked
+                                                           ? question.numberOfEntitiesToGiveFeedbackTo
+                                                           : 1);
```

Comment:
This is better as it was before, i.e don't add another level of indentation for the `?` and `:`.